lib/kms.js

```diff
@@ -13,22 +13,27 @@ const {
   KeyAgreementKey,
   KmsClient
 } = require('webkms-client');
-const {util: {clone, BedrockError}} = bedrock;
+const {config, util: {clone, BedrockError}} = bedrock;
+
+const cfg = config.profile;
 
 exports.createKeystore = async ({
-  capabilityAgent, kmsBaseUrl, referenceId
```

Comment:
So I think it's fine to omit this for now. But remember that a future use case is enabling users to pick a different KMS service -- which would only apply to the so-called "public keystores". The "private keystores" (the ones with the IP allow list applied) would still use whatever the application has configured for its own internal use. I think we'd also have to pass a zcap to enable the application to do the keystore creation on behalf of the user -- so there would be more needed anyway. We'll figure out what changes need to happen when we implement the feature for allowing users to pick their own KMS for "public keystores" in the future -- this is good for now.